NewForce-Cohort-5/tabloidmvc-the-daily-debuggers
# Bg user profile details

# Description

When you click users at the top of the page and click the eye for user details, you should see all of the relevant details of that user.  A default image is used for any user profile that doesn't contain an image link already [Click here to see it](https://avatars.dicebear.com/api/bottts/.svg).

## Type of change


# Testing Instructions

1. Clone down the repository.
2. `git fetch -a`
3. `git checkout bg-UserProfileDetails`
4. Navigate to the root directory for this repo (directory-to)/tabloidmvc-the-daily-debuggers
5. Start TabloidMVC.sln
6. Make sure your db is created with seed data. Run [01_TabloidMVC_Create_DB.sql](https://github.com/NewForce-Cohort-5/tabloidmvc-the-daily-debuggers/blob/main/SQL/01_TabloidMVC_Create_DB.sql) and [02_TabloidMVC_Seed_Data.sql](https://github.com/NewForce-Cohort-5/tabloidmvc-the-daily-debuggers/blob/main/SQL/02_TabloidMVC_Seed_Data.sql) queries in Visual Studio.
7. Start the server.
8. Log in as admin [admin@example.com](mailto:admin@example.com) (if not already logged in as admin).
9. Click Users at the top of the page
10. You should see a list of all users.
11. Click the eye next to a user (You should have at least one user if you used the seed data from Step 4)
12. You will see the details of the users profile you clicked including an image.  A default image is used if ImageLocation is Null.



# Checklist:
